lib/cmd/common/version.js

```diff
@@ -120,26 +112,5 @@ module.exports = {
     }
     return util.format(i18n._('Warning - newer FHC version available (%s). To update, run\n' +
       'npm install -g fh-fhc'), this.latest.version);
-  },
-  //Checking If The Platform Version Can Be Used With This Version Of fhc
-  checkTargetVersion: function(targ, cb) {
```

Comment:
Not sure if this was working but represents valid use case (telling users if their client needs update). 
Going to research how this was done and let you know.